### DISCUSSION TITLE
WARNING Server sent a  mtprotostate.py:216   very new    message with ID

### DISCUSSION THREAD

**Discussion by Sl0thNet** [aahnik/tgcf#493] [General]
Hello. I've been up and running with tgcf for about a week on VPS Ubuntu Server 20.04. It stopped forwarding messages and started printing the following error: WARNING Server sent a mtprotostate.py:216 very new message with ID 726848091165266 1249, ignoring. I am attaching a log file. I couldn't solve the problem, so I'm writing here. Please tell me where I am wrong.
Thanks in advance!
[tgcf_logs.txt](https://github.com/aahnik/tgcf/files/12448623/tgcf_logs.txt)

**Comment by ghost** [comment]
Could be the time set wrong on your VPS or session used in other place. [Source](https://stackoverflow.com/questions/75933350/telethon-telegram-lib-giving-me-server-sent-a-very-new-message-with-id).

**Reply by Sl0thNet** [reply]
I use the same time zone for the VPS server and for telegram.

**Reply by ghost** [reply]
Set the time to automatically adjust.

**Reply by Sl0thNet** [reply]
I think the time is set correctly.
![2023-08-29_22h53_20](https://github.com/aahnik/tgcf/assets/143033683/0cdd13a5-2115-422c-9215-5dde91a96768)

**Reply by ghost** [reply]
If is set manually, try to set it to automatically get the time. If that is not working, try to delete the old session string and generate a new one.
